modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -449,6 +459,8 @@ def test_sort_multiindex(sort_remaining):
 def test_sort_values(
     data, by, axis, ascending, inplace, kind, na_position, ignore_index, key
 ):
+    if ascending is None:
```

Comment:
How does pandas process this parameter?

Reply:
This is not an allowed value for `ascending` anymore.

Reply:
ok